department-of-veterans-affairs/va.gov-cms
# VACMS-4844: Fix broken events view.

## Description
See #4844 

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/113447811-9a83c000-93c8-11eb-92f5-876c1c3a3be5.png)

## QA steps
As an administrator, go to `/admin/config/system/taxonomy-entity-index/reindex` and reindex content
Once complete, visit `/admin/content/events` and verify view is working as expected.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
